pkg/ibmvpc/analysis_output_test.go

```diff
@@ -470,6 +470,11 @@ var tests = []*vpcGeneralTest{
 		useCases:    []vpcmodel.OutputUseCase{vpcmodel.EndpointsDiff},
 		format:      vpcmodel.Text,
 	},
+	{
+		inputConfig: "acl_testing3",
+		useCases:    []vpcmodel.OutputUseCase{vpcmodel.EndpointsDiff},
+		format:      vpcmodel.JSON,
+	},
```

Comment:
for the `diff` command we do not yet officially support the JSON format (I recall there could be some open issue with relevant questions about this output) .

Reply:
ok, removed.

Reply:
I see there are in fact json testing @adisos 
```
	// json examples
	{
		inputConfig: "demo_with_instances",
		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints, vpcmodel.AllSubnets},
		format:      vpcmodel.JSON,
	},
	{
		inputConfig: "acl_testing3",
		useCases:    []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
		format:      vpcmodel.JSON,
	},
```